app/Http/Controllers/Api/UsersController.php

```diff
@@ -505,22 +505,11 @@ public function emailAssetList(Request $request, $id)
         $user = User::findOrFail($id);
 
         if (empty($user->email)) {
-            return response()->json(
-                [
-                    'status' => 'error',
-                    'message' => 'This user has no email set.',
-                    'payload' => null,
-                ], 404);
+            return response()->json(Helper::formatStandardApiResponse(‘error’, $accessory, trans('admin/accessories/message.inventorynotification.error')));
```

Comment:
I'm not sure we need `$accessory` here either, since this isn't specifically in the context of an accessory. (I'm not even sure how that wouldn't kick back an error, since I don't think `$accessory` is defined yet in that method.) Maybe pass it `$user` or `null` there?

Reply:
Honestly, we may not. Upon testing, there is something else that supercedes this line, and greys out the "email list of all assigned" box. So, I wasn't able to specifically test the error.

Reply:
I think we just need to use `null` there instead of `$accessory`. 

To test this, use Postman and trigger `/api/v1/users/{userId}/email` - replacing `{userId}` in the endpoint URL with the ID of a user that does not have an email address.